a3p-integration/proposals/n:upgrade-next/test.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -euo pipefail
+
```

Comment:
please include an XXX about these named tests. It's tech debt because it should be a wildcard. This hard-coded list risks omitting a test.

The comment should warn the reader that they should verify this list against the directory contents and updated the list whenever a test is added.

Reply:
I switched to using wild card invocation, with an exception for `initial.test.js`.